.clang-format

```diff
@@ -36,7 +36,7 @@ BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Linux
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: false
-ColumnLimit:     85
+ColumnLimit:     100
```

Comment:
I'm not very fond of way more than 80 because half of my laptop screen fits 80. Also the width on GitHub seems to favor something 80-ish. 100 seems to be too large for GitHub such that one has scroll bars there. So I would prefer a smaller value.